CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ Change Log
 * Fixed links in URL lookup search results that needed encoding. [#118](https://github.com/unt-libraries/django-nomination/issues/118)
 * Converted fielded_batch_ingest.py script to a Django management command and improved speed.
 * Updated info about bookmarklets. [#120](https://github.com/unt-libraries/django-nomination/issues/120)
+* Remove use of global variables for tracking batch nominations.
```

Comment:
Does this really need to be mentioned if it was only in for a short period of time?

Reply:
How long is long enough for a change to be logged :D? I might rather justify taking it out since it is a change to implementation that shouldn't be noticeable in use.

Reply:
Gotcha. It's removed.